dbos/cli/cli.py

```diff
@@ -41,7 +41,7 @@ def _on_windows() -> bool:
     help="Start your DBOS application using the start commands in 'dbos-config.yaml'"
 )
 def start() -> None:
-    config = load_config()
+    config = process_config(data=load_config())
```

Comment:
I hate to add more complexity to this PR, but I've been thinking about how these commands (particularly `migrate` and `debug` don't work well when using the new `DBOSConfig` mode. I think the solution is to add a `database_url` field to `dbos-config.yaml` so you can pass a connection string in there (perhaps through an environment variable). That way if you want to use `dbos migrate` or the debugger, you simply set the field to whatever environment variable you're loading your connection string from.

Reply:
Why wouldn't they work? Both need the config file. Users can put their db connection info in there.

Are you saying that, for ease of use, you'd rather have the file able to load a connection string, so they can use the same (constructed) environment variable in code and in CICD ?

Reply:
Yes